File: prog69.py
#!/bin/python3
def main():
    while True:
        try:
            n1 = int(input("Enter a number: "))
            break
        except ValueError:
            print("You must enter an integer. Please try again.")
    print("Double it and give it to the next person: ", n1 * 2)
if __name__ == "__main__":
    main()

File: prog69.r
main <- function() {
  while(TRUE) {
    n1 <- readline(prompt="Enter a number: ")
    if (!is.numeric(n1)) {
      cat("You must enter an integer. Please try again.\n")
    } else {
      break
    }
  }
  cat("Double it and give it to the next person: ", as.numeric(n1) * 2, "\n")
}

main()
